setRemoteOperationListener is not documented (#1750)
microsoft/appcenter-sdk-apple: CHANGELOG.md

### App Center Data

* **[Fix]** Fix an issue where invalid characters in the document ID are accepted at creation time but causing errors while trying to read or delete the document. The characters are `#`, `\`, `/`, `?`, and all whitespaces.

> * **[Feature]** `setRemoteOperationListener` method allows to be notified of a pending operation being executed when a client device goes from offline to online.

Reviewer: ```suggestion
* **[Feature]** `setRemoteOperationDelegate` method allows to be notified of a pending operation being executed when a client device goes from offline to online.
```

> * **[Feature]** `setRemoteOperationDelegate` method allows to be notified of a pending operation being executed when a client device goes from offline to online.

Reviewer 2: ```suggestion
* **[Feature]** `setRemoteOperationDelegate` method to set a delegate to be notified of a pending operation being executed when a client device goes from offline to online.
```

> * **[Feature]** `setRemoteOperationDelegate` method to set a delegate to be notified of a pending operation being executed when a client device goes from offline to online.

Reviewer 3: ```suggestion
* **[Feature]** Added `setRemoteOperationDelegate` method to set a delegate to be notified of a pending operation being executed when the client device goes from offline to online.
```
